share/translations/translations.pro

```diff
@@ -36,7 +36,7 @@ include(../../src/libs/vtools/vtools.pri)
 include(../../src/libs/vformat/vformat.pri)
 include(../../src/libs/fervor/fervor.pri)
 
-# Add here path to new translation file with name "seamly2d_*_*.ts" if you want to add new language.
+# Add here path to new translation file with name "measurements_*_*.ts" if you want to add new language.
```

Comment:
My bad...  I was trying to restore the pro files and must have mixed the up. Will fix as soon as I can.

Reply:
Apparently pulling the changes in PR 880  messed up the translations.pro. It's correct on my end. ?? As soon as my flakey internet comes back I'll fix it.

Reply:
Translations.pro should be fixed. 

Just a note: Initially I removed the 2 translation project files believing they were RT hacks and not  to be use as per his comments:

# Don't use this pro file for building project!!!
# I made this hack for creation one TS file for whole project.
# Because we have several libraries that also have strings for translation, have several TS files for each
# subproject annoying.
#
# Original idea "How to generate a single translation file for a large Qt project?"
# https://stackoverflow.com/questions/1829474/how-to-generate-a-single-translation-file-for-a-large-qt-project

As the files are indeed needed I suggest at some point we remove the contradicting comments.